### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by HubertGrabda**

### CODE REVIEW

File: Twittfake/src/components/Navbar/Navbar.jsx [hunk 1]
```diff
@@ -7,34 +7,17 @@ import {
...
-
-    return () => {
-      window.removeEventListener("scroll", handleScroll);
-    };
-  });
+  const showElement = ScrollHandler();
 
   return (
-    <div className={`navbar ${showNavbar ? "" : "hidden"}`}>
+    <div className={`navbar${showElement ? "--hidden" : ""}`}>
```
Comment (PatrykGrabda) [inline Twittfake/src/components/Navbar/Navbar.jsx]: If `showElement` is `true` then it rather shouldn't have `--hidden` class

Comment (PatrykGrabda) [inline Twittfake/src/components/Navbar/Navbar.jsx]: U can also check `classNames` from ddtvn-waiter repo, `helper.js`. We're using this one for handling classnames which depends on variables/props. 

```
export function classNames(array) {
    return array.filter((value) => value).join(' ');
}
```

example of use: 
```
<figure
            className={classNames([
                'stories-list-content',
                'stories-list-article-content',
                isPwa && 'stories-list-content--pwa',
            ])}>
            <Link href={url}>
            [...]
```

Comment (HubertGrabda) [reply Twittfake/src/components/Navbar/Navbar.jsx]: done

Comment (HubertGrabda) [reply Twittfake/src/components/Navbar/Navbar.jsx]: done

**Pull request merged**